rules/linux/auditd/lnx_auditd_pers_systemd_reload.yml

```diff
@@ -4,23 +4,30 @@ status: test
 description: Detects a reload or a start of a service.
 references:
     - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1543.002/T1543.002.md
-author: Jakob Weinzettl, oscd.community
+author: Jakob Weinzettl, oscd.community, CheraghiMilad
 date: 2019-09-23
-modified: 2021-11-27
+modified: 2024-11-30
 tags:
     - attack.persistence
     - attack.t1543.002
 logsource:
     product: linux
     service: auditd
 detection:
-    selection:
+    selection_systemctl:
         type: 'EXECVE'
         a0|contains: 'systemctl'
         a1|contains:
             - 'daemon-reload'
             - 'start'
-    condition: selection
+    selection_service:
+        type: 'EXECVE'
+        a0|contains: 'systemctl'
+        a1|contains:
+            - 'start'
+            - 'reload'
+            - 'restart'
+    condition: 1 of selection_*
```

Comment:
Hi,
type and a0 are the same, `start` is duplicate in this case
Why not simply add 'reload' and 'restart' to the selection ?

Reply:
Thank you for your attention. Unfortunately, I had a typo, which has been corrected. In fact, services in Linux can be manipulated in various ways, and I tried to add the service to that.
However, daemon-restart has a significant false-positive issue in Linux because the kernel manipulates services through the daemon, and this is often a natural behavior.